api_confluence.py

```diff
@@ -208,9 +226,91 @@ def pages():
                                    current_version, new_content)
             page_payload_write(page_id, payload)
 
+# ------------------------------------------------------------------
+# Page rendering: XML configs 
+# ------------------------------------------------------------------
+
+def render_xml_template(template_path, params):
+    """ 1. open Mustache-style (jinja compatible) XML template 
+        2. parameterize template with params """
+    with open(template_path, 'r') as file:
+        template_content = file.read()
+    
+    template = Template(template_content)
+    rendered_xml = template.render(**params)
+    
+    return rendered_xml
+
+
+def page_report_build_validation():
+    """
+    iterates over confluence XML files and generates pages
+    """
+
+    # TODO: hard-coded params for now...
```

Comment:
Testing functionality on the Demo 2 page.  This code path will not be activated by the github action until phase 2